contracts/root/predicates/ERC20Predicate.sol

```diff
@@ -504,11 +525,9 @@ contract ERC20Predicate is IErcPredicate {
   {
     bytes4 funcSig = BytesLib.toBytes4(BytesLib.slice(txData, 0, 4));
     if (funcSig == WITHDRAW_FUNC_SIG) {
-      require(txData.length == 36, "Invalid tx"); // 4 bytes for funcSig and a single bytes32 parameter
       amount = BytesLib.toUint(txData, 4);
       exitType = ExitType.Burnt;
     } else if (funcSig == TRANSFER_FUNC_SIG) {
```

Comment:
`require(txData.length == 36, "Invalid tx");` these checks were for collision attacks with function sig(due to smaller size 4bytes), will need to be always carefull with new contracts additions.

Reply:
Assertions on length aren't doing us much good.

Reply:
yeah, although we should be careful with new token mappings.